draft-ietf-rats-eat.md

```diff
@@ -223,32 +221,15 @@ CWT and JWT.
 
 # Introduction
 
-Some of the goals and fundamentals in the security model for attestation are not the same as other security standards such as those aimed at privacy (e.g., TLS) and authentication (e.g. FIDO).
-The security model for attestation is not described here.
-Instead see {{RATS.Architecture}}.
+The Entity Attestation Token (EAT) format enables entities to create EATs, which are attested claims sets
+that describe characteristics of the entity, i.e., a device, a software environment, a hardware environment or combination. The EAT format provides a common set of claims that can be used to represent evidence and attestation results. A relying party may use an EAT when determining the trustworthiness of the corresponding entity, and may choose to trust, not trust or partially trust the entity. An entity that verifies an EAT may create an EAT to convey attestation results for consumption by other entities.
 
-EAT provides the definition of a common set of claims that can be made about an entity, a device, some software and/or some hardware.
-This claims set is received by a relying party who uses it to decide if and how it will interact with the remote entity.
-It may choose to not trust the entity and not interact with it.
-It may choose to trust it.
-It may partially trust it, for example allowing monetary transactions only up to a limit.
+The EAT format defines the encoding of claims sets in both CBOR {{RFC8949}} and JSON {{RFC8259}} and defines EATs as extensions to the CBOR Web Token (CWT) {{RFC8392}} and JSON Web Token (JWT) {{RFC7519}} specifications. Authenticity and integrity protection MUST be provided and privacy (encryption) MAY additionally be provided for EATs. The CBOR Object Signing and Encryption (COSE) {{RFC9052}} and JSON Object Signing and Encryption (JOSE) {{RFC7515}} {{RFC7516}} SHOULD be used to provide these security services. Terminology, security models and operational models defined in [RATS.architecture] are used throughout.
```

Comment:
The MUST and SHOULDs in this paragraph provide little value since they are so unspecific. I think you can get away with just using "should" and "must" in the introduction.

Reply:
Changed.